packages/admin/cms-admin/src/generator/future/generateForm.ts

```diff
@@ -16,13 +17,12 @@ export function generateForm(
     const gqlQueries: Record<string, string> = {};
     const imports: Imports = [];
 
+    const fieldNamesFromConfig = config.fields.map<string>((field) => field.name);
```

Comment:
@johnnyomair solved it by adding <string>. Hovering over field.name said type is string. so this was probably some typescript bug.

Reply:
regarding the question in slack, how I should fix the typescript issue with recommended refactor using for...of instead of map vs ts-ignore.

Reply:
this is also solved 👍